EspaceNetworks/SuiteCRM
# SMTP Keep Alive for Campaign Mass Emailer.

## Description



Activate SMTP Keep Alive for Campaign Mass Emailer.

## Motivation and Context

To reduce the CPU load on the SMTP Server, and reduce the chance of errors and throttling of the Campaign Mass Emailer by improving performance.

## How To Test This

Create a Campaign in a version of SuiteCRM without this update.
Create a Campaign and send it using the Scheduler.
Calculate the time for each batch by viewing the log and search for the timestamps and subtract start time from end time for each batch of X emails (default is 500 emails).
Then run a Campaign on a version of SuiteCRM with this patch.
Send it using the Scheduler.
Calculate the time for each each batch by viewing the log, and search for the timestamps and subtract start time from end time for each batch of X emails (default is 500 emails).
The time to send the batch of campaign emails should be significantly faster.

## Types of changes


### Final checklist
